src/Stan/Report/Html.hs

```diff
@@ -41,334 +40,318 @@ import Stan.Severity (Severity (..), severityDescription)
-        # div_ (severityFromIns ins)
-        # div_ (categories "" inspectionCategory)
-        # solutionsDiv ins
-        )
-    )
+stanInspections :: HashSet (Id Inspection) -> Html
+stanInspections ins = do
+    divClass "row" (blockP "List of Inspections used for analysing the project")
+    div $ sequence_ $ map stanInspection $ sortWith unId $ toList ins
+
+stanInspection :: Id Inspection -> Html
+stanInspection (getInspectionById -> ins@Inspection{..}) = do
+    button ! A.class_ "collapsible" ! A.id (fromString $ toString insId) $
+      toHtml ("Explore Inspection " <> insId)
+    divClass "content row" $ divIdClass (toString insId <> "-content") "inspection col-12" $ do
+        h3 $ toHtml ("Inspection " <> insId)
+        p $ strong $ toHtml inspectionName
+        p $ em $ toHtml inspectionDescription
+        div (severityFromIns ins)
+        div (categories "" inspectionCategory)
+        solutionsDiv ins
   where
     insId :: Text
     insId = unId inspectionId
 
-stanConfig Analysis{..} (config :: Config) (warnings :: [Text]) = divClass "col-12 "
-    ( divClass "row" (blockP "Description of the custom Stan configuration and explanation of how it was assembled")
-    # divClass "row" (table_
-        ( tr_ (th_ "Action" # th_ "Filter" # th_ "Scope")
-        # map toRows (configToTriples config)
-        ))
-    # divClass "ignored-observations row"
-        ( toUl ignoredIds "Ignored Observations"
+stanConfig :: Analysis -> Config -> [Text] -> Html
+stanConfig Analysis{..} config warnings = divClass "col-12" $ do
+    divClass "row" (blockP "Description of the custom Stan configuration and explanation of how it was assembled")
+    divClass "row" $ table $ do
+        tr (th "Action" >> th "Filter" >> th "Scope")
+        sequence_ $ map toRows (configToTriples config)
+    divClass "ignored-observations row" $ do
+        toUl ignoredIds "Ignored Observations"
             "These observations are flagged as ignored through the configurations and are not considered in the final report"
-        # toUl unknownIds "Unrecognised Observations"
+        toUl unknownIds "Unrecognised Observations"
             "Some observation IDs specified in the configurations are not found"
-        )
-    # divClass "config-warnings row"
-        ( h4_ "Configuration Process Information"
-        # p_
-            ( "Information and warnings that were gathered during the configuration assemble process. "
-            # "This helps to understand how different parts of the configurations were retrieved."
-            )
-        # ul_ (map li_ warnings)
-        )
-    )
+    divClass "config-warnings row" $ do
+        h4 "Configuration Process Information"
+        p $
+            "Information and warnings that were gathered during the configuration assemble process. "
+          <> "This helps to understand how different parts of the configurations were retrieved."
+        ul (sequence_ $ map (li . toHtml) warnings)
```

Comment:
I think I saw a similar couple of times... Probably can have a function like :slightly_smiling_face: 

```haskell
uList :: ToMarkup a => [a] -> Html
```